### PR TITLE
Add another version of homework2

### PR DESCRIPTION
**Pull request by n-gb**

### CODE REVIEW

**Review by oleksandr-kychun — COMMENTED** [review]
The most interesting implementation of object_model so far)
About state_and_behavior:
It is preferable to use hsh.fetch to get values and to provide defaults. Use getter/setter instead of changing instance_variables directly.

File: lib/topics/state_and_behaviour.rb [hunk 1]
```diff
@@ -2,4 +2,29 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # The class we work with
+  class Car
+    attr_accessor :year, :color, :model, :current_speed
+
+    def initialize(args)
```
Comment (sveredyuk) [inline lib/topics/state_and_behaviour.rb:9]: args of what?

Comment (sveredyuk) [reply lib/topics/state_and_behaviour.rb:9]: and try to add default data, like args = {}

Comment (n-gb) [reply lib/topics/state_and_behaviour.rb:9]: If I add default data it fails one of the tests - "expects hash of attributes". In other words, it stops raising ArgumentError.

**Pull request closed without merging**